locations/spiders/rubios.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+
+from locations.items import GeojsonPointItem
+
+
+class RubiosSpider(scrapy.Spider):
+    name = "rubios"
+    allowed_domains = ['rubios.com']
+    start_urls = (
+        'https://www.rubios.com/sitemap.xml',
+    )
+
+    def parse(self, response):
+        response.selector.remove_namespaces()
+        city_urls = response.xpath('//url/loc/text()').extract()
+        for path in city_urls:
+            regex = re.compile(r'http\S+rubios.com/store-locations/\S+/\S+/\S+')
+            if not re.search(regex,path):
+                pass
+            else:
+                yield scrapy.Request(
+                    path.strip(),
+                    callback=self.parse_store,
+                )
+
+    def parse_store(self, response):
+
+        if response.css('.store-info').extract_first().replace('\t', '').split('<span class="oh-display-label" style="width: 6.6em;">')[1].split('<br></span>')[0].replace('</span><span class="oh-display-times oh-display-hours">', '').strip() is not None or '':
+            properties = {
+            'name': response.css('.store-info').extract_first().replace('\t','').split('<span itemprop="name">')[1].split('</span>')[0],
```

Comment:
Try your hardest to not use splits as they're pretty inefficient and brittle (it's what's causing the exception in the log). It would be better to use `xpath()` here like this:

```
'name': response.xpath('//span[@itemprop="name"]/text()').extract_first(),
...
'telephone': response.xpath('//span[@itemprop="telephone"]/a/text()').extract_first(),
```